leveling/rank.cc.lua

```diff
@@ -5,13 +5,19 @@
 
 	Recommended trigger: Regex trigger with trigger `\A(-|<@!?204255221017214977>\s*)(rank|lvl|xp)(\s+|\z)`.
 */}}
+{{/* CONFIGURATION AREA STARTS */}}
+{{ $rankcard := 1 }} {{/* set to `1` for a rank card */}}
```

Comment:
Please change this to 0 (or even better, use a boolean like a normal person) to keep the previous behavior and make this opt-in for now.

Reply:
> Please change this to 0 (or even better, use a boolean like a normal person) to keep the previous behavior and make this opt-in for now.

but im not normmal okay :(